fog/test-client/src/test_client.rs

```diff
@@ -676,37 +870,85 @@ impl TestClient {
         log::debug!(self.logger, "Creating {} clients", client_count);
         let clients = self.build_clients(client_count);
 
-        log::debug!(self.logger, "Generating and testing transactions");
-
-        let start_time = Instant::now();
-        for ti in 0..num_transactions {
-            log::debug!(self.logger, "Transation: {:?}", ti);
+        // Send test transfers in each configured token id
+        for token_id in &self.policy.token_ids {
+            log::debug!(
+                self.logger,
+                "Generating and testing {} transactions",
+                token_id
+            );
 
-            let source_index = ti % client_count;
-            let target_index = (ti + 1) % client_count;
-            let source_client = clients[source_index].clone();
-            let target_client = clients[target_index].clone();
+            let start_time = Instant::now();
+            for ti in 0..num_transactions {
+                log::debug!(self.logger, "Transation: {:?}", ti);
+
+                let source_index = ti % client_count;
+                let target_index = (ti + 1) % client_count;
+                let source_client = clients[source_index].clone();
+                let target_client = clients[target_index].clone();
+
+                let transaction = self.test_transfer(
+                    token_id.clone(),
+                    source_client.clone(),
+                    source_index,
+                    target_client,
+                    target_index,
+                )?;
```

Comment:
Potentially add another test with slightly more complex swap list?

Reply:
yeah we could do that -- i think it's worth just getting the basic functionality to work first